src/main/scala/mesosphere/marathon/api/v2/json/Formats.scala

```diff
@@ -239,6 +238,14 @@ trait EventFormats {
     )
   }
 
+  lazy val LightDeploymentPlanWrites: Writes[DeploymentPlan] = Writes { plan =>
+    Json.obj(
```

Comment:
As @jdef noted in a comment: can you please use raml and have the formats for this and the other messages generated?

Reply:
+1 Please introduce RAML types.